Menus/MetagameGUI.h

```diff
@@ -438,6 +438,11 @@ class MetagameGUI:
 // Return value:    None.
 	void SetToStartNewGame();
 
+	/// <summary>
+	/// Sets where the station is located on the planet orbit.
+	/// </summary>
+	/// <param name="newStationPos">The position of the station on the planet orbit.</param>
+	void SetStationInfo(const Vector &newStationPos) { m_StationPosOnOrbit = newStationPos; }
```

Comment:
`SetStationPos` would be misleading because it doesn't actually set the station position, just passes it on to be used elsewhere.
Also why you @ ing random people.

Reply:
lol dunno how I typo'd in an @ there.

I guess you're saying that the m_StationPosOnOrbit is just read by something else? Still, it's setting a variable named m_StationPosOnOrbit, not m_StationInfo. If you wanna call it `SetStationOrbitPos` or something else, I don't mind that, but SetStationInfo isn't exactly informative, and it reads like someone renamed the variable but forgot to rename the method or something.